packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -514,6 +521,11 @@ export class GarbageCollector implements IGarbageCollector {
             }
         }
 
+        // Sweep V0 overrides Sweep Timeout, even setting it if SessionExpiry is disabled
+        if (this.sweepV0TimeoutMs !== undefined) {
+            this.sweepTimeoutMs = this.sweepV0TimeoutMs;
```

Comment:
Should this be a Math.min() ? Just in case one value is much larger than another one.